Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -135,6 +135,8 @@
     <string name="wiimote_speaker_description">Enable sound output through the speaker on a real Wiimote (DolphinBar required).</string>
     <string name="audio_stretch">Audio Stretching</string>
     <string name="audio_stretch_description">Stretches audio to reduce stuttering. Increases latency.</string>
+    <string name="analytics">Analytics</string>
```

Comment:
The word "analytics" shouldn't be used in user-facing strings. We've decided on "usage statistics reporting" instead.